app/src/main/java/com/T05/krowdtrialz/model/user/User.java

```diff
@@ -0,0 +1,4 @@
+package com.T05.krowdtrialz.model.user;
+
+public class User {
+}
```

Comment:
This conflicts with @Jacques857's changes in #55

Reply:
Well, not conflict, but there are two copies of User class

Reply:
My changes in #55 simply create a placeholder User class so that my ContributorList array adapter class for the ListView in the contributors screen doesn't have a bunch of errors. The only methods I added were a constructor and a getter for the name attribute. If you have a class that already does this it would be fine to just erase my User class.